Tasks/UPackCommandV0/Common/ArtifactToolRunner.ts

```diff
@@ -0,0 +1,68 @@
+import * as auth from "./Authentication";
+import * as tl from "vsts-task-lib/task";
+let fs = require("fs");
+let os = require("os");
+import child = require("child_process");
+import stream = require("stream");
+import {IExecOptions, IExecSyncResult} from "vsts-task-lib/toolrunner";
+
+export interface IArtifactToolOptions {
+    artifactToolPath: string;
+    feedId: string;
+    accountUrl: string;
+    packageName: string;
+    packageVersion: string;
+    authInfo: auth.UPackExtendedAuthInfo;
```

Comment:
Where is authInfo used?

Reply:
It's not used now. I can remove it. I put it there initially to support username/password.